library/bignum.c

```diff
@@ -116,6 +120,7 @@ void mbedtls_mpi_free( mbedtls_mpi *X )
 int mbedtls_mpi_grow( mbedtls_mpi *X, size_t nblimbs )
```

Comment:
I'd say 'no'; definitely 'no' for a parameter validation check, and 'perhaps' for an unconditional check, but even in that case, I'm inclined to leave it, because the function behaves correctly if you pass `0`.

Reply:
On reflection, I don't think we should change it.